### ISSUE TITLE
Issue with large/old OnlyFans Models

### ISSUE THREAD

**Issue by gitporn873** [DIGITALCRIMINAL/OnlyFans#458]
When scraping models with a lot of content (my example is rileyreidx3 ) the script just sorta hangs there in a loop. I've let it run for 2 days straight before and same issue. This seems to be related to more recent updates as this model used to scrape just fine.

Output from the script is just the newer `FOUND: /whatever/OnlyFans/rileyreidx3/Posts/Free/Images` message that outputs.

Has anyone else encountered this?

Also wondering if there's a way to supress these messages as printing to screen does slow scripts down and as a user I'd prefer to only get errors and informational type messages.

**Comment by James-MF-Thomas** [comment]
I dont know her content, but Ive been  able to download multiple models with >2000 vids per account. one model had >8000 photos. completed fine.   found string just means that directory is already on your drive. try sending to new location maybe

**Comment by gitporn873** [comment]
If I send to another location it's gonna grab all the old files too though. Used to not be an issue but seems to be now. Just seems strang that the found string gets printed thousands of times.

From my last run on this Model she has 5734 photos and about 300 videos. The script seems to never even get past the images no matter how long I let it run

**Comment by James-MF-Thomas** [comment]
you can set config file to loop infinity after completion or for an x amount of time. not sure if that could be relevant to your situation.  

(if you transfer the photos on to a new location with the same name, it should still register the no_overwrite command)

**Comment by gitporn873** [comment]
I'm only having it run once, no loops

**Comment by DIGITALCRIMINAL** [comment]
Yeah, Riley Reid just naturally takes forever. Are you using a proxy?

**Comment by gitporn873** [comment]
Nah no proxy just straight through. Any way you know of that can speed up her dump? I know in another issue you said it's hard to use the current metadata file to get the most recently archived media and use that as an "end date" sort of timestamp so I doubt it.

Would it be a massive codebase overhaul to implement that sort of functionality? I'm not the best Python dev but if it's nothing that changes the fundamental architecture of the code I might be down to look over the code this weekend or something and see if I can implement that sort of check

**Comment by DIGITALCRIMINAL** [comment]
> Nah no proxy just straight through. Any way you know of that can speed up her dump? I know in another issue you said it's hard to use the current metadata file to get the most recently archived media and use that as an "end date" sort of timestamp so I doubt it.
> 
> Would it be a massive codebase overhaul to implement that sort of functionality? I'm not the best Python dev but if it's nothing that changes the fundamental architecture of the code I might be down to look over the code this weekend or something and see if I can implement that sort of check

Yes, I'd have to overhaul everything, but I'm redoing pretty much everything anyway since the script looks like a mess. I wanna make it clearer for me and others to read and edit.

**Comment by gitporn873** [comment]
Gotcha, I'll leave you to that and close this issue then. Good luck on the overhaul and thanks for the awesome project.

**Comment by DIGITALCRIMINAL** [comment]
Ima reopen this so I don't forget
